RecaptchaValidation/Services/RecaptchaService.cs

```diff
@@ -47,57 +44,51 @@ public async Task<IRecaptchaResponseMessage> Execute()
                 
                 if (!Response.success)
                 {
-                    throw new RecaptchaRequestException();
+                    throw new RecaptchaRequestException($"Errors returned from Recaptcha Verification URL: {Response.error_codes}");
```

Comment:
If you wanted to, you could build this string into the RecaptchaRequestException and just pass in the error codes when you throw a new instance of the exception.

The `RecaptchaRequestException` could then utilize the error codes when returning its message.

Can do some research into custom exception creation and how to extend their behavior outside of merely changing the type name.